packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -280,29 +294,32 @@ async function validateGossipAttestationNoSignatureCheck(
   const attEpoch = computeEpochAtSlot(attSlot);
   const attTarget = attData.target;
   const targetEpoch = attTarget.epoch;
+  let committeeIndex;
+  if (attestationOrCache.attestation) {
+    if (isElectraAttestation(attestationOrCache.attestation)) {
+      // api or first time validation of a gossip attestation
+      const {committeeBits} = attestationOrCache.attestation;
+      if (committeeBits == null) {
```

Comment:
use ===

Reply:
should throw both in case of undefined and null, commended that